utils/client.py

```diff
@@ -34,6 +34,7 @@
     'sponsor': ['sponsor', (), (), (), ()],
     'sponsors': ['sponsors', (), (), (), ()],
     'team': ['teams', ('', 'id'), ('name', 'creator_id', 'hidden'), ('id',), ('id',)],
+    'fcmtoken': ['fcmtokens', ('', 'id'), ('fcm_token','fbid',), ('id',), ('id',)],
```

Comment:
Please also add the client.py functions for notification and participant-notification.

Reply:
Added additional functions